up42/utils.py

```diff
@@ -111,7 +112,7 @@ def download_from_gcs_unpack(
             directory.
     """
     # Download
-    out_temp = tempfile.mkstemp()[1]
+    out_temp = tempfile.mkstemp(dir=output_directory)[1]
```

Comment:
Coverage is missing

Reply:
I had to rename one test that was misnamed. I added tests for the previously missing function and a new one to check that the temp file was in the output_directory.